dist-material/release-docs/NOTICE

```diff
@@ -838,7 +838,43 @@ from and not be held liable to the user for any such damages as noted
 above as far as the program is concerned.
 
 ------
+===========================================================================
+nacos-1.3.0 Notice
+===========================================================================
+Nacos
+Copyright 2018-2019 The Apache Software Foundation
+
+This product includes software developed at
+The Alibaba MiddleWare Group.
```

Comment:
@songzhendong If you want to keep the NOTICE(we could keep). We just need the above part. 
Please take a look the latest version, I have sent a PR to fix it. 

The things below we don't use, so, don't need to add.